src/combat/rqgToken.ts

```diff
@@ -28,4 +34,26 @@ export class RqgToken extends Token {
       });
     }
   }
+
+  //TODO: When upgrading @league-of-foundry-developers/foundry-vtt-types
+  //we can remove the ts-nocheck at the top of this file.
+  //In the meantime this allows us to get the userId which is now the third
+  //parameter.  We need it to ensure that we don't initialize the unlinked
+  //Actor for every logged in user.
```

Comment:
I agree that it's problematic with the old Typescript types. My plan was to skip the intermediate versions and go directly to Foundry 10 version. Unfortunately the velocity of that project has dropped since the main developer has quit. There still is some movement so I'm still hoping for a usable version.

In the meantime I think it's better to use expect error like this and not have @ts-nocheck since those will just be forgotten about, while the @ts-expect-error will be an error when the ts types "fit again". 
```suggestion
  // @ts-expect-error 
```

That expect error is also needed on the CONFIG row.